### ISSUE TITLE
Operator "<=" not supported for types "Timestamp" and "TimestampSeries"

### ISSUE THREAD

**Issue by timovp** [pandas-dev/pandas-stubs#27]
Hi all, 
I raised an issue over at pylance: 
https://github.com/microsoft/pylance-release/issues/2933
These has been an update, but the error became TimestampSeries instead Series[Timestamp]
to replicate: 
```python
import pandas as pd

df =  pd.DataFrame(['2020-01-01','2019-01-01'])
dates = pd.to_datetime(df[0], format = '%Y-%m-%d')
date_to_compare = pd.to_datetime('2019-02-01', format = '%Y-%m-%d')
date_mask = date_to_compare <= dates
```
Generated the type error in vs code: 

> Operator "<=" not supported for types "Timestamp" and "TimestampSeries"
  Operator "<=" not supported for types "Timestamp" and "TimestampSeries"
  Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pylance-release/blob/main/DIAGNOSTIC_SEVERITY_RULES.md#diagnostic-severity-rules)

Would like to create a PR to help, but just started to figure out how type stubs work, so might need a time :-)

**Comment by Dr-Irv** [comment]
@timovp To encourage you to do the PR, here is what you'd need to do.  Look in `series.pyi` and the overloads for `__add__`.  You'll need to do something similar for `__lt__`, `__le__`, `__gt__`, `__ge__`, except the results in all cases are `Series[bool]`

**Comment by timovp** [comment]
created a PR #28 for it. thanks again for the encouragement!

**Comment by Dr-Irv** [comment]
closed in #28
